benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -23,8 +23,7 @@
 from benchmark.opperf.rules.default_params import DEFAULTS_INPUTS, MX_OP_MODULE
 
 # Operators where parameter have special criteria that cannot be cleanly automated.
-# Example: sample_multinomial operator has a parameter 'data'. It expects values to sum up to 1.
-unique_ops = ("sample_multinomial",)
+unique_ops = ()
```

Comment:
since this is empty. do we need it any longer?

Reply:
Good point, we do not! Will remove.